app/install.php

```diff
@@ -132,7 +132,7 @@ function saveStep2() {
 		$config_array = [
 			'salt' => generateSalt(),
 			'base_url' => $base_url,
-			'default_user' => 'admin',
+			'default_user' => '_',
```

Comment:
So that we can *touch* e.g. an SQLite database in the `./data/users/_/` folder, which is the only one we have so early in the install process